lib/jxl/render_pipeline/stage_write.cc

```diff
@@ -38,23 +38,48 @@ using hwy::HWY_NAMESPACE::VFromD;
 // 8x8 ordered dithering pattern from
 // https://en.wikipedia.org/wiki/Ordered_dithering
 // scaled to have an average of 0 and be fully contained in (-0.5, 0.5).
-const float kDither[64] = {
-    -0.4921875, 0.0078125,  -0.3671875, 0.1328125,   //
-    -0.4609375, 0.0390625,  -0.3359375, 0.1640625,   //
-    0.2578125,  -0.2421875, 0.3828125,  -0.1171875,  //
-    0.2890625,  -0.2109375, 0.4140625,  -0.0859375,  //
-    -0.3046875, 0.1953125,  -0.4296875, 0.0703125,   //
-    -0.2734375, 0.2265625,  -0.3984375, 0.1015625,   //
-    0.4453125,  -0.0546875, 0.3203125,  -0.1796875,  //
-    0.4765625,  -0.0234375, 0.3515625,  -0.1484375,  //
-    -0.4453125, 0.0546875,  -0.3203125, 0.1796875,   //
-    -0.4765625, 0.0234375,  -0.3515625, 0.1484375,   //
-    0.3046875,  -0.1953125, 0.4296875,  -0.0703125,  //
-    0.2734375,  -0.2265625, 0.3984375,  -0.1015625,  //
-    -0.2578125, 0.2421875,  -0.3828125, 0.1171875,   //
-    -0.2890625, 0.2109375,  -0.4140625, 0.0859375,   //
-    0.4921875,  -0.0078125, 0.3671875,  -0.1328125,  //
-    0.4609375,  -0.0390625, 0.3359375,  -0.1640625,  //
+// Matrix is duplicated in width to avoid inconsistencies or out-of-bound-reads
+// if doing unaligned operations.
+const float kDither[64 * 2] = {
```

Comment:
I think this would make it more obvious how it conforms to the comment:

```suggestion
const float kDither[(2 * 8) * 8] = {
```

Reply:
done (both)